docs/approach-docs/README.md

```diff
@@ -21,3 +21,4 @@ The existing types are:
 
 - `system-` for a `system` or set of conventions
 - `feature-` for creating a new or enhancing an existing `feature`
+- `platform-` for infrastructure or process level improvements
```

Comment:
Not sure this is the most appropriate name, `infrastructure` was my other opinion

Reply:
Good point! I think `infrastructure` probably fits best with the terminology we often use but platform works fine as well!